youthful-piety/modernized/12-chapter-11.adoc

```diff
@@ -968,20 +978,26 @@ many times I have not been in a fit state to receive good; at others,
 when I have been uncomfortable or under discouragement,
 some beautiful passages have been presented, which were very consoling,
 such as '`The Lord is my Shepherd, I shall not want;
-he makes me to lie down in green pastures, he leades me beside the still waters.`"
+he makes me to lie down in green pastures, he leads me beside the still waters.`"
 I will never leave you nor forsake you.`'`"
 
 About a week afterwards, writing to a friend, she says,
```

Comment:
This is where the letter starts in this paragraph and continues on to another one. I thought it looked good like this and I probably have more of an inclination to want to embed things than not, but totally up to you if you wanna change it!